lib/spree/auth/engine.rb

```diff
@@ -46,10 +46,6 @@ def self.activate
         ApplicationController.send :include, Spree::AuthenticationHelpers
       end
 
-      def self.api_available?
```

Comment:
Duplicate of https://github.com/spree/spree_auth_devise/blob/a0c0fee800063cfae52c0a2c888eed0dc9be02b1/lib/spree/auth/engine.rb#L57